src/main/java/org/terasology/metalrenegades/PlayerInventorySystem.java

```diff
@@ -72,19 +67,9 @@
 
     @Override
     public void postBegin() {
+        // Ensure that CombatSystem does not spawn its own starting inventory.
```

Comment:
Can this whole method be removed? What does `CombatStartingInventory` do now that it does not need to do anything?

Reply:
As it currently stands, ```CombatStartingInventory``` runs for any module that has ```CombatSystem``` as a dependency. If a module that depends on ```CombatSystem``` does not change the starting inventory then the inventory specified in ```CombatSystem``` will be added regardless.  

As a test you can run this branch and comment out the entire ```MetalRenegades:PlayerInventorySystem.postBegin()``` method, and you should see that the ```CombatSystem``` inventory is added <em>in addition</em> to the ```MetalRenegades``` starting inventory.

I hope I'm making sense.

Reply:
Absolutely! I now did understand that you're actually resetting the starting inventory which would have contained combat items to not have any items initally. 
The previous code did overide the default starting inventory of CombatSystem with what MetalRenegades has.

The whole code block can be removed once we removed the default starting inventory from CombatSystem module (and move it to a command or some other gameplay module), is that correct?

Reply:
Yes, I believe that's correct.